packages/react/src/ErrorBoundaryGroup.tsx

```diff
@@ -28,14 +28,14 @@ export const ErrorBoundaryGroup = ({
   children?: ReactNode
 }) => {
   const [resetKey, reset] = useKey()
+  const parentGroup = useContext(ErrorBoundaryGroupContext)
+  const isChangedParentGroupResetKey = useIsChanged(parentGroup?.resetKey)
 
-  const isMounted = useIsMounted()
-  const group = useContext(ErrorBoundaryGroupContext)
   useEffect(() => {
-    if (isMounted && !blockOutside) {
+    if (!blockOutside && isChangedParentGroupResetKey) {
       reset()
     }
-  }, [group?.resetKey, isMounted, reset])
+  }, [isChangedParentGroupResetKey, reset, blockOutside])
```

Comment:
<img width="271" alt="image" src="https://github.com/suspensive/react/assets/69495129/6468da86-612e-4c44-8f6f-810aff3a9221">

I think it can't be done for deleting line  (I knew today 😆)
<img width="271" alt="image" src="https://github.com/suspensive/react/assets/69495129/626cb56b-71dd-4d6e-9a49-6298241828b3">

Reply:
Ahha, Okay!

Reply:
I renamed `isChangedParentGroupResetKey` to `isParentGroupResetKeyChanged` and added you as co-author in [d9cd0a8](https://github.com/suspensive/react/pull/92/commits/d9cd0a8c44f0a5456919e19d49ec6b83bbf01b25)